frontend/src/containers/Home.css

```diff
@@ -1,3 +1,22 @@
 .usa-search__submit-text {
   vertical-align: text-top !important;
 }
+
+.cards ul {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 0;
+}
+
+.cards ul li {
+  list-style: none;
+  flex: 0 0 33.333333%;
+}
+
+.divbackground {
+  background-color: rgb(220, 220, 220);
```

Comment:
it is recommended to use colors from the theme palette. Rename file to .scss and use 

```sass

.divbackground {
   u-bg("white");
};

```

some import may be missing

Reply:
Check the flex items in the USWDS theme, that may help to fix the issue without us writing custom code

https://designsystem.digital.gov/design-tokens/flex/

Reply:
Switched to the theme palette.